docs-source/content/faq/coherence-requirements.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Coherence Requirements"
+date: 2019-08-12T12:41:38-04:00
+draft: false
+---
+
+If you are running Coherence on Kubernetes, either inside a WebLogic domain
+or standalone, then there are some additional requirements to make sure
+that Coherence can form clusters. 
+
+Note that some Fusion Middleware products, like SOA Suite, use Coherence
+and so these requirements apply to them.
+
+#### Unicast and Well Known Address 
+When the first Coherence process starts, it will form a cluster.  The next
+Coherence process to start (i.e. in a different pod) will use UDP to try
+to contact the master.  Most Kubernetes overlay network providers do not
+support multicast, so you should configure Coherence to use unicast and 
+provide a "well known address" so that members can find the master. 
+This is normally done by using the cluster address, i.e. the address of
+the Kubernetes service that is pointing to all cluster members.
+
+{{% notice note %}}
+If you create your domain using the samples provided in this project, the
+cluster will be able to form, you do not need to manually specify WKA.
+However, the information is included below for completeness.
+{{% /notice %}}
+
+For example, suppose you have a domain called `domain1` and in that domain
+you have a cluster called `cluster1` and Coherence is running in that
+cluster.  You would therefore use the cluster service that is created by
+the operator as the well known address.  This service, in this example, 
+would be called `domain1-cluster-cluster1` and follows the pattern:
+
+```
+domainUID  "-cluster-"  cluster name
```

Comment:
The operator normalizes domainUID and cluster-name when it converts them to an address in order to make them DNS compatible (all lower case, underscores converted to dashes).

Reply:
done